src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -447,6 +452,53 @@ private void testEditNonExistentQuestion() {
         // Restore feedbackEditPage
         feedbackEditPage = getFeedbackEditPage();
     }
+    
+    private void testResponseRate() {
+        
+        // Create a new question and save
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.fillQuestionBox("new question");
+        feedbackEditPage.clickAddQuestionButton();
+
+        // Create response for the new question
+        FeedbackResponseAttributes feedbackResponse = 
+                new FeedbackResponseAttributes(
+                        feedbackSessionName,
+                        courseId, 
+                        "1", 
+                        FeedbackQuestionType.TEXT,
+                        "tmms.test@gmail.tmt", 
+                        Const.DEFAULT_SECTION, 
+                        "alice.b.tmms@gmail.tmt",
+                        Const.DEFAULT_SECTION, 
+                        new Text("Response from instructor to Alice"));
+        BackDoor.createFeedbackResponse(feedbackResponse);
+
+        ______TS("check response rate before editing question");
+        
+        AppUrl feedbacksPageUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE).withInstructorId(instructorId);
+        InstructorFeedbacksPage feedbacksPage = AppPage.getNewPageInstance(browser, feedbacksPageUrl, InstructorFeedbacksPage.class);
```

Comment:
Extract this into a new method. Also, check if using <code>AppPage#changePageType</code> is more suitable here.